packages/mds-policy-service/service/provider.ts

```diff
@@ -48,7 +48,12 @@ export const PolicyServiceProvider: ServiceProvider<PolicyService> & ProcessCont
   readSinglePolicyMetadata: policy_id =>
     serviceToPolicyRepository('readSinglePolicyMetadata', () => PolicyRepository.readSinglePolicyMetadata(policy_id)),
   readBulkPolicyMetadata: params =>
-    serviceToPolicyRepository('readSinglePolicyMetadata', () => PolicyRepository.readBulkPolicyMetadata(params)),
+    serviceToPolicyRepository('readBulkPolicyMetadata', () => {
+      if (params.get_unpublished && params.get_published)
+        throw new BadParamsError('cannot have get_unpublished and get_published both be true AHHHHH')
```

Comment:
This seems a little funky, couldn't you want to query for the metadata for both published and unpublished policies at the same time? Or in that case, would you just provide neither parameter?

If we really do want to throw an error in this case, should probably make the error message a little less fun, this will percolate to API clients :)

Reply:
I think it's fine to throw an error, rather than re-interpret the intent of the caller to mean "both" and pass neither attribute to the query engine. 

The AHHHHH part was my fault! I was debugging, trying to figure out where the error handling was falling down in this case, and "AHHHH" is easy to search for in the logs.